### PR TITLE
Add field validator id for validators table.

### PR DESCRIPTION
**Pull request by Benzbeeb**
## Implementation details

1. Add field id in validators table.
2. Change all foreign key that refers from operator address to id.
3. Fix flusher handler

---

Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: flusher/flusher/db.py [hunk 1]
```diff
@@ -147,14 +147,14 @@ def Column(*args, **kwargs):
     "val_requests",
     metadata,
     Column("request_id", sa.Integer, sa.ForeignKey("requests.id"), primary_key=True),
-    Column("validator", sa.String, sa.ForeignKey("validators.operator_address"), primary_key=True),
+    Column("validator", sa.Integer, sa.ForeignKey("validators.id"), primary_key=True),
```
Comment (sorawit) [inline flusher/flusher/db.py]: Let's call all of these "validator_id"

**Review by sorawit — CHANGES_REQUESTED** [review]

File: flusher/flusher/db.py [hunk 2]
```diff
@@ -147,14 +147,14 @@ def Column(*args, **kwargs):
     "val_requests",
     metadata,
     Column("request_id", sa.Integer, sa.ForeignKey("requests.id"), primary_key=True),
-    Column("validator", sa.String, sa.ForeignKey("validators.operator_address"), primary_key=True),
+    Column("validato_id", sa.Integer, sa.ForeignKey("validators.id"), primary_key=True),
```
Comment (sorawit) [inline flusher/flusher/db.py]: This won't work. Please make sure to test it.

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]

File: flusher/flusher/handler.py [hunk 3]
```diff
@@ -73,35 +79,46 @@ def handle_new_raw_request(self, msg):
...
         )
 
     def handle_update_validator(self, msg):
-        condition = True
-        for col in validators.primary_key.columns.values():
-            condition = (col == msg[col.name]) & condition
-        self.conn.execute(validators.update().where(condition).values(**msg))
+        self.conn.execute(
+            validators.update()
+            .where(msg["operator_address"] == validators.c.operator_address)
```
Comment (taobun) [inline flusher/flusher/handler.py]: swap operand side

**Review by taobun — CHANGES_REQUESTED** [review]

File: flusher/flusher/db.py [hunk 4]
```diff
@@ -202,13 +203,8 @@ def Column(*args, **kwargs):
 delegations = sa.Table(
     "delegations",
     metadata,
+    Column("validator_id", sa.Integer, sa.ForeignKey("validators.id"), primary_key=True),
     Column("account_id", sa.Integer, sa.ForeignKey("accounts.id"), primary_key=True),
```
Comment (taobun) [inline flusher/flusher/db.py]: Revisit all account_id.

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**